packages/components/src/components/modal/modal-styles.ts

```diff
@@ -238,6 +243,7 @@ export const getComponentCss = (
         left: '8px',
         display: 'flex',
         justifyContent: 'flex-end',
+        zIndex: 1, // To be on top of transformed full screen content
```

Comment:
Comment is somewhat wrong or at least confusing.
I guess it is related to full width and `stretchToFullModalWidthClassName`?